db/migrate/20220202141625_create_users.rb

```diff
@@ -1,7 +1,7 @@
 class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
-      t.string :name, null: false
+      t.string :name, null: false #TODO: Validação da parte do banco. com index e uniq
```

Comment:
Aqui não é o melhor lugar para deixar esse `TODO`, uma vez que o PR for mergeado na `main`, a gente não deve mais mexer na migration (ela será rodada no banco de produção e não vai existir a possibilidade de fazer um rollback), sendo necessário criar uma nova migration com essa regra adicional aí.

Como no caso é a tabela de usuários (que não precisa de escopo), uma mudança assim já resolveria:

```suggestion
      t.string :name, null: false, unique: true
```